server/service/src/processors/transfer/shipment/create_inbound_shipment.rs

```diff
@@ -135,10 +135,10 @@ fn generate_inbound_shipment(
         picked_datetime: outbound_shipment_row.picked_datetime,
         shipped_datetime: outbound_shipment_row.shipped_datetime,
         transport_reference: outbound_shipment_row.transport_reference.clone(),
+        comment: outbound_shipment_row.comment.clone(),
```

Comment:
the mSupply version has the comment from a `their_ref` field. Just checking: we aren't going to do that?

```
	[transact]comment:=Get indexed string(17014; 210)+" "+Choose([transact]comment=""; ""; "("+[transact]comment+")")  // Stock transfer + comment
	[transact]their_ref:=Get indexed string(17014; 209)+": "+$t_serial+Choose([transact]their_ref=""; ""; " ("+[transact]their_ref+")")  // "From invoice number: " + Serial number + "Their ref" field, if entered
```

Reply:
I can add them in with the comment, must have missed the comments in some PR (: !

Reply:
[b611083](https://github.com/openmsupply/open-msupply/pull/664/commits/b6110831ad4ad7c712cbabb0b1238e0ffd8ca470)